Update c_anc_eff1-3(_q)

hzyguan888 opened the pull request:
Please check and let me know if you agree. Because previously, c_anc_eff and c_anc_eff_q have the same observations, which is not correct

hzyguan888 committed:
Update c_anc_eff1-3(_q) 

Please check and let me know if you agree. Because previously, c_anc_eff and c_anc_eff_q have the same observations, which is not correct

wengxyu1030 commented on 1_antenatal_care.do:
```diff
@@ -47,7 +47,7 @@ order *,sequential
 	*c_anc_eff: Effective ANC (4+ antenatal care visits, any skilled provider, blood pressure, blood and urine samples) of births in last 2 years
 	egen anc_blood = rowtotal(m42c m42d m42e),mi
 	gen c_anc_eff = (c_anc == 1 & anc_skill>0 & anc_blood == 3) 
-	replace c_anc_eff = . if c_anc ==. |  anc_skill==. | anc_blood ==.
```
The definition of c_anc_eff changed? So blood pressure is not considered in the effective ANC?

hzyguan888 replied:
No change of definition. Because there are filter questions for blood pressure, we cannot say if anc_blood=. then c_anc_eff =.

wengxyu1030 replied:
The missing is for the cases where: All non-missing conditions “YES” and 1 or more missing answers on any of the conditions missing OR time of pregnancy missing. 

So anc_blood is a condition?

wengxyu1030 commented on 1_antenatal_care.do:
```diff
@@ -130,19 +130,22 @@ order *,sequential
+	
 	*c_anc_eff3: Effective ANC (4+ antenatal care visits, any skilled provider, blood pressure, blood and urine samples, tetanus vaccination, start in first trimester) of births in last 2 years 
 	gen c_anc_eff3 = (c_anc == 1 & anc_skill>0 & anc_blood == 3 & rh_anc_neotet == 1 & inrange(m13,0,3)) 
-	replace c_anc_eff3 = . if c_anc == . | anc_skill == . | anc_blood == . | rh_anc_neotet == .
```
What is rh_acnc_neotet?

hzyguan888 committed:
Update c_anc_eff-3(_q)

Fixed the filter question for anc_blood. c_anc_eff should have similar observations as c_anc now.

wengxyu1030 committed:
Merge pull request #17 from wengxyu1030/hzyguan888-patch-9

Update c_anc_eff-3(_q)
